app/extensions/safe/utils/safeHelpers.js

```diff
@@ -24,5 +24,9 @@ export const urlIsAllowed = ( testUrl ) =>
         return true;
     }
 
+    if(urlObj.protocol === 'blob:') {
```

Comment:
@hunterlester let's add this above with the standard protocol inclusion (the localhost whitelist one's are separate for clarity)

Reply:
@joshuef Let me know if the change I made is what you had in mind.